gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs/api-runtime-logs.component.spec.ts

```diff
@@ -170,6 +171,108 @@ describe('ApiRuntimeLogsComponent', () => {
     });
   });
 
+  describe('GIVEN there are logs with filters', () => {
+    beforeEach(async () => {
+      await initComponent();
+    });
+
+    describe('when we arrive at default state', () => {
+      const total = 1;
+      beforeEach(() => {
+        expectApiWithLogs(total);
+        fixture.detectChanges();
+        expectApiWithLogEnabled();
+      });
+
+      it('should display the 1st page with default pagination', async () => {
```

Comment:
Maybe we could change the test description. Finally the most important thing here is that the periodSelectInput value is none by default. You could even remove line 193 and 194 as it's already covered in a previous test I think